Oculo/HealthkitManager.swift

```diff
@@ -152,23 +155,23 @@ class HealthKitManager {
         healthStore.execute(query)
     }
 
-    // 센치미터 단위로 입력했을 때 예상 걸음 수 반환하는 메서드
-    // ex. 500센치 거리 입력, 유저의 보폭이 75일 때 500/75 == 6.66666666667
-    //   => 6걸음 이상 걸어야 하니깐 안전을 위해 6걸음으로 내림해서 리턴
+    /// 센티미터 단위 거리를 받아  예상 걸음 수 반환 메서드
+    /// ex. 500센치 거리 입력, 유저의 보폭이 75일 때 500/75 == 6.66666666667
+    ///   => 6걸음 이상 걸어야 하니까 안전을 위해 6걸음으로 내림해서 리턴
     public func calToStepCount (centimeter:Double) -> Int {
         return Int(floor(centimeter/Double(self.walkingStepLength)))
     }
 
-    // 미터 단위로 입력했을 때 예상 걸음 수 반환하는 메서드
+    /// 미터 단위로 입력했을 때 예상 걸음 수 반환 메서드
     public func calToStepCount (meter:Double) -> Int {
```

Comment:
```suggestion
    public func calToStepCount(meter: Double) -> Int {
```
이 부분도 나중에 같이 수정하면 좋을 것 같습니다!

Reply:
> 이 부분도 나중에 같이 수정하면 좋을 것 같습니다!

반영했습니다. 미처 다 잡지 못했네용!